server/web.js

```diff
@@ -250,6 +302,21 @@ exports.createFrontend = function createFrontend(config, db) {
       res.send(historical);
     });
   });
+
+  // TODO: Better way to take in the cutoff than as a '?cutoff='' ?
```

Comment:
Does this make sense with no cutoff? For a live view, seems like we would be re-requesting *all* ops after every op? Live view should use a stream of ops from sharedb if we want one, and perhaps this URL always requires a cutoff.

Reply:
And then you can use the same pattern as `historical`, with cutoff at the end.